core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -123,13 +122,12 @@ export class VisualizationPanelContentComponent implements AfterContentInit, OnD
     if (!this.operatorID) {
       return;
     }
-    const result: ResultObject | undefined = this.workflowStatusService.getCurrentResult()[this.operatorID];
-    if (!result) {
+    const opratorResultService = this.workflowResultService.getResultService(this.operatorID);
```

Comment:
I still think this `Service` name does not make sense here. Can we change it to something simple like `const operatorResult = this.workflowResultService.getOperatorResult(this.operatorID)`?

Reply:
done

Reply:
I thought about this again and I don't like the `operatorResult` name because this object is managing the result. 

There are a couple other names to choose: `manager`, `controller`, but I like the name `service` here. Also don't be `service` isn't necessarily always a singleton. See the stackoverflow discussion: https://stackoverflow.com/questions/8403535/when-do-i-call-my-class-controller-manager-or-service

Therefore I'd like to stick to the service name.